includes/import/abstract-wc-product-importer.php

```diff
@@ -726,4 +726,22 @@ protected function explode_values( $value ) {
 	protected function explode_values_formatter( $value ) {
 		return trim( str_replace( '::separator::', ',', $value ) );
 	}
+
+	/**
+	 * The exporter prepends a ' to fields that start with a - which causes
+	 * issues with negative numbers. This removes the ' if the input is still a valid
+	 * number after removal.
+	 *
+	 * @since 3.3.0
+	 * @param string $value A numeric string that may or may not have ' prepended.
+	 * @return string
+	 */
+	protected function unescape_negative_number( $value ) {
+		$unescaped = str_replace( "'-", '-', $value );
```

Comment:
This could use some improvement because it will replace at any point in the string. We only want to remove if it's at the beginning of the string.